silverback/cluster/client.py

```diff
@@ -168,13 +166,13 @@ def health(self) -> BotHealth:
         return BotHealth.model_validate(raw_health)  # response.json())  TODO: Migrate this endpoint
 
     def stop(self):
-        response = self.cluster.post(f"/bots/{self.id}/stop")
+        response = self.cluster.post(f"/bots/{self.name}/stop")
```

Comment:
this endpoint does not exist, do we want a stop endpoint?

Reply:
I think we had a stop endpoint before, it would basically stop the container from running

It doesn't necessarily have to deprovision it because Silverback has a stop/idle sequence, but would be best for idempotency if it deprovisioned as well